packages/theme/stories/introduction.stories.mdx

```diff
@@ -208,27 +202,27 @@ The palette token then resolves down to:
+Note that the CSS variable is the same - the values associated with it differ between the light and dark `scopes`. When this variable is used by a Button to style the button text, the color applied will be appropriate to the theme variant applied to the page. Because CSS variable scopes can be nested, there is great flexibility for combining regions with different styling variants within the same app. If a page with a light theme requires that a smaller region of that page be styled with a dark or even custom theme, that is straightforward.
 
-### Component styles
+#### Component styles
 
 Component styles work with theme styles to give a complete theme-based styling solution. At the component level, we specify exactly which HTML elements have particular styles applied to them. By generalising these styles across the application, which is what the theme variables allow us to do, we achieve consistency of design across the our application and across applications. This is the Design System concept in operation.
-We have already mentioned that components will reference Theme variables, the example below is from Button. It shows just a single style declaration but illustrates some important concepts:
+We have already mentioned that components will reference Theme variables, the example below is from Button. It shows a single style declaration but illustrates some important concepts:
 
 ```css
 .uitkButton {
   font-size: var(--uitkButton-fontSize, var(--uitk-text-fontSize));
 }
 ```
 
-1. All UI Toolkit components have a CSS class on the root element, of the form `uitk<ComponentName>`. This is a rule with no exceptions.
-2. The value from a CSS API variable (more on this below) is assigned as a style by using `var(--<variableName>-<...>);`.
-3. A CSS variable reference can define a fallback value to be used if the primary variable does not have a value:
-   var(--primary-variable-value, var(--fallback-value))
-4. The fallback value can equally be a simple constant CSS value
-   var(--primary-variable-value, 12px);
+- All UI Toolkit components have a CSS class on the root element, of the form `uitk<ComponentName>`. This is a rule with no exceptions.
+- The value from a CSS API variable (more on this below) is assigned as a style by using `var(--<variableName>-<...>);`.
+- A CSS variable reference can define a fallback value to be used if the primary variable does not have a value:
+  var(--primary-variable-value, var(--fallback-value))
```

Comment:
There is an inconsistency where backticks are used for CSS vars in some but not all cases. Plus, I think we are missing some full stops here.

Reply:
Thanks, fixed